src/tool/cli.rs

```diff
@@ -61,6 +61,7 @@ impl Cli {
 pub enum Commands {
     #[clap(visible_alias = "gen", short_flag = 'g')]
     Generate(GenerateArgs),
+    Copy,
```

Comment:
Maybe we should add a short note explaining what the command does. (In hindsight, I think I will add one for generate as well.)
```suggestion
     /// Copies the code from ??? to your clipboard for pasting on leetcode
    Copy,
```

Reply:
I planned to do this in a later PR, adding such comments to all commands, subcommands, flags, etc.

Reply:
I see, let's include one for Copy for now as I needed it to understand what the expected behaviour is.